### ISSUE TITLE
Hi, I'm wondering about comparative test.

### ISSUE THREAD

**Issue by DBpackage** [dmis-lab/PerceiverCPI#4]
Hi, I was really impressed with your paper. It was really nice and well-explained. Particularly, I was impressed with the introduction section  which includes the drawbacks of the existing approaches and the contributions which your model has.

<img width="868" alt="image" src="https://github.com/dmis-lab/PerceiverCPI/assets/68315582/0fdb697f-0269-4e83-b859-762671ab7e04">

BTW, here, about the comparative test, I'm wondering how did you do the test on the transformerCPI model.
During the step of doing the test on the BindingDB dataset on transformerCPI, (don't mind about the DB. I just want to know how to train with dataset which transformerCPI team doesn't provide on the github page) I couldn't do the training with bindingDB dataset since their pre-processing methods and training models are different among the Datasets(e.g Kinase, GPCR, Humans/C.Elegans, BindingDB, They gave all distinct codes for each DB)

And I couldn't find the DAVIS dataset on the transformerCPI github which you did it. 
So I'm asking how did you do the preprocessing on the DAVIS dataset and how did you train the transformerCPI model with DAVIS. (I mean, which code you used in the transformerCPI github, there are various models and preprocessing codes for distinct DB) 

Thanks in advance,
Again, I respect your work, sincerely

**Comment by quangnguyenbn99** [comment]
Hello,

Regarding the transformerCPI model experiments, as transformerCPI was originally designed for binary classification problems, we made modifications to the data processing from the source code to adapt it for a regression problem. Specifically, our experiments were conducted using data from the GPCR folder, and the process unfolded as follows:
1 The CSV data was converted to TXT format, excluding headers.
2 The 'mol_featurizer.py' script was executed to create the model input.
3 The loss function was switched from CrossEntropyLoss to MSELoss.
It's worth noting that all other aspects of the original architecture remained unchanged.

Thank you for your inquiry.

**Comment by DBpackage** [comment]
Thanks for replying,

But I'm still confused. :(

I thought you used YOUR OWN DAVIS dataset which you give the link in the supplementary.
<img width="867" alt="image" src="https://github.com/dmis-lab/PerceiverCPI/assets/68315582/8b4c7db3-5e5c-4ca1-8cf3-10b4dd3ae567">


<img width="907" alt="image" src="https://github.com/dmis-lab/PerceiverCPI/assets/68315582/d143712d-cc7c-4d28-9a70-0d329dc44b57">
In above table, you did comparative test with DAVIS on transformerCPI.

My question is how did you do the test with DAVIS dataset on transformerCPI? (not for the GPCR dataset)

(Maybe I'm confused with this sentence. 
"Specifically, our experiments were conducted using data from the GPCR folder"
here using data from the GPCR folder means literally GPCR dataset? or mol_featurizer.py and main.py from GPCR folder?)

TransformerCPI gives us GPCR dataset and mol_featurizer.py and main.py for it but they didn't give us the mol_featurizer.py and main.py for DAVIS dataset so I can't do the fair test with DAVIS dataset on transformerCPI.

Are you meaning that you've just conducted the test with your DAVIS on GPCR mol_featurizer.py and main.py?

I'm sorry that I'm bothering you.
I really appreciate your time and consideration.

**Comment by quangnguyenbn99** [comment]
Hello,
I am sorry for the confusing answer. I meant that:
              We modified the mol_featurizer.py to make it applicable to all types of datasets. 
Yes, we conduct experiments on mol_featurizer.py, main.py and model.py (changing the last layer and loss function)

**Comment by DBpackage** [comment]
Thanks for your answer!

I got it now :)
